sdk/confidentialledger/Azure.Security.CodeTransparency/README.md

```diff
@@ -0,0 +1,120 @@
+# Azure Code Transparency client library for .NET
+
+<!-- cspell:ignore cose merkle scitt -->
+
+`Azure.Security.CodeTransparency` is based on a managed service complying with a [draft SCITT RFC][SCITT_ARCHITECTURE_RFC]. It is a managed service that allows countersigning [COSE signature envelopes][COSE_RFC]. Countersignatures are recorded and signed in the immutable merkle tree for any auditing purposes and [the receipt][SCITT_RECEIPT_RFC] gets issued.
+
+- [OSS server application source code][Service_source_code]
+
+## Getting started
+
+### Install the package
+
+Make sure you have access to the correct Nuget Feed.
+
+Install the client library for .NET with [NuGet](https://www.nuget.org/ ):
+
+```dotnetcli
+dotnet add package Azure.Security.CodeTransparency --prerelease
+```
+
+### Prerequisites
+
+- A running and accessible Code Transparency Service
+- Ability to create `COSE_Sign1` envelopes, [an example script][CTS_claim_generator_script]
+- Your signer details (CA cert or DID issuer) have to be configured in the running service, [about available configuration][CTS_configuration_doc]
+- You can get a valid Bearer token if the service authentication is configured to require one, [see example]<!--(https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/confidentialledger/Azure.Security.CodeTransparency/samples/Sample3_UseYourCredentials.md)-->
```

Comment:
Unless something recently changed, since we're targeting main with the merge and the references trace back to main, we should be able to uncomment them all here without upsetting the analyze leg of CI.

Leaving one comment here, but the samples also follow the same pattern.

Reply:
I do not believe this is true.  Unless I am misremembering, the analyzer will detect that this is a PR that targets main and evaluate the state of `main` that would occur if merged.